db is up-to-date
Creating djangoapp ... 
Creating djangoapp ... done
[Pipeline] }
[Pipeline] // stage
[Pipeline] stage
[Pipeline] { (Run Tests)
[Pipeline] sh
+ docker-compose exec -T djangoapp python manage.py test
[Pipeline] }
[Pipeline] // stage
[Pipeline] stage
[Pipeline] { (Run Django Setup)
Stage "Run Django Setup" skipped due to earlier failure(s)
[Pipeline] }
[Pipeline] // stage
[Pipeline] }
[Pipeline] // withEnv
[Pipeline] }
[Pipeline] // node
[Pipeline] End of Pipeline
ERROR: script returned exit code 1
Finished: FAILURE
